pkg/binlog/position.go

```diff
@@ -202,6 +206,16 @@ func (l Location) Clone() Location {
 	}
 }
 
+// CloneGTIDSet clones location's gtid set
+func (l Location) CloneGTIDSet() gtid.Set {
```

Comment:
It seems not used now.

Reply:
removed in /run-all-tests tidb=release-3.0